pkg/asset/userprovided.go

```diff
@@ -30,7 +32,10 @@ func (a *UserProvided) Generate(map[Asset]*State) (*State, error) {
 }
 
 // QueryUser queries the user for input.
-func QueryUser(inputReader *bufio.Reader, prompt string) string {
+func QueryUser(inputReader *bufio.Reader, prompt, defaultValue string) string {
+	if defaultValue != "" {
+		prompt = strings.Join([]string{prompt, "[", defaultValue, "]"}, "")
```

Comment:
`strings.Join` is a surprising way to do this (although it should work fine).  I'd have expected:

```go
prompt := fmt.Sprintf("%s [%s]", prompt, defaultValue)`
```

or similar (maybe `%s [default: %s]`?).

Reply:
On another day I would have definitely done fmt.Sprintf. Not sure what sprang into my mind today :). Will normalize.

Reply:
+1